#! /bin/sh /usr/share/dpatch/dpatch-run
## bts-383368_fix_warning.dpatch by Sandro Tosi <matrixhasu@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: bts-383368_fix_warning

@DPATCH@
diff -urNad xsysinfo~/xsysinfo.c xsysinfo/xsysinfo.c
--- xsysinfo~/xsysinfo.c	2006-08-16 22:59:17.000000000 +0300
+++ xsysinfo/xsysinfo.c		2006-08-16 22:58:52.000000000 +0300
@@ -27,6 +27,7 @@
 #include <X11/Xaw/Label.h>
 #include "BarGauge.h"
 #include <stdio.h>
+#include <stdlib.h>
 #include "sysinfo.h"

 #define norm(x,max)    ((max) == 0 ? 0 : (double)(x) / (max))
@@ -325,9 +326,7 @@
	fprintf(stderr,"\nor one of the standard toolkit options.\n\n");
 }

-main( argc, argv )
-	int argc;
-char **argv;
+int main( int argc, char **argv )
 {
	Arg args[1];
	Widget toplevel;
@@ -358,4 +357,5 @@

	XtRealizeWidget(toplevel);
	XtAppMainLoop(context);
+	return 0;
 }
